pagopa/mui-italia
# feat(PN-9570): button visibility adaptation to standard WCAG 2.2

## Short description
Adapt button visibility on focus-visible to WCAG 2.2 standards.

### Preview
![image](https://github.com/pagopa/mui-italia/assets/73442810/facce4ae-aa4d-4066-8d68-0c64b09ec555) ![image](https://github.com/pagopa/mui-italia/assets/73442810/8215332d-4808-4470-9e73-f9916b8fb68d) ![image](https://github.com/pagopa/mui-italia/assets/73442810/94d0ccf4-8670-454e-abe7-35109173ae1b) ![image](https://github.com/pagopa/mui-italia/assets/73442810/79eea6de-9e53-4336-a799-1aca9461b152) ![Screenshot 2024-04-11 alle 09 17 07](https://github.com/pagopa/mui-italia/assets/73442810/65e41d0d-9291-454a-b1a2-b1e99a44f9d1)

## List of changes proposed in this pull request
- Add outline to buttons and links
- Add padding to date-picker icons

## Product
Piattaforma Notifiche -> SEND

## How to test
Run storybook and check Button, Button Naked, Link and IconButton have outline of 2px
